programmers/두개-뽑아서-더하기.js

```diff
@@ -0,0 +1,11 @@
+function solution(numbers) {
+    let newArr = [];
+    for (let i = 0; i < numbers.length + 1; i += 1 ) {
+        for (let j = i + 1; j < numbers.length; j += 1) {
+            newArr.push(numbers[i] + numbers[j])
+        }
+    }
+    let arrdpRmvd = new Set(newArr);
+    console.log([...arrdpRmvd]);
+    return [...arrdpRmvd].sort((a, b) => a < b ? -1 : 0);
+}
```

Comment:
```suggestion
function solution(numbers) {
    const newArr = [];
    
    for (let i = 0; i < numbers.length + 1; i += 1) {
        for (let j = i + 1; j < numbers.length; j += 1) {
            newArr.push(numbers[i] + numbers[j])
        }
    }
    
    return [...new Set(newArr)].sort((a, b) => a - b);
}
```
set를 따로 선언안하고 해결가능합니다.
지난번에도 말한 let보다 const선언이 좋아요 (메모리 낭비시간 줄일 수 있답니다)
console,log() 제거는 기본 ~